src/code.html

```diff
@@ -38,7 +38,7 @@ <h3>Project templates</h3>
     </main>
     <footer>
       <nav aria-label="Primary">
-        <a href="/src/sitemap.html">Contents</a>
+        <a class="local" href="/src/sitemap.html">Contents</a>
```

Comment:
Did you pull main first? We removed the local class so that Home and Contents are more visible.

Reply:
Also, I'd like to clear the backlog of PRs before moving on to avoid merge issues.

Reply:
I thought I did pull main first. I'm don't remember what we agreed on in terms of the local class for Home and Contents. Whatever works best for you, let me know and I'll make the changes.

For the git Actions PR, I will work on it hopefully tomorrow. I'm sure it's fine, I just want to read the article before merging.

Reply:
We agreed to remove the local class so that Home and Contents stands out.

You'll need to make the changes manually because the merge will think your changes override the current state.

Best not done while drinking champagne (although that might make an entertaining piece). Bom ano!

Reply:
`git remote prune origin` is handy for getting latest branches on origin especially after a PR clean up has deleted one.

There's also git switch e.g. `git switch -c new-branch` or simply `git switch existing-branch` as alternatives to using `git checkout`. But it's a matter of personal preference.